tests/test_http.py

```diff
@@ -119,6 +119,9 @@ def test_cache_control_header(self):
         c.max_age = 3.1
         assert c.max_age == 3
         del c.max_age
+        c.max_age = 3
+        assert c.max_age == 3
+        del c.max_age
```

Comment:
These test should be `c.s_maxage`.  `max_age` in 2.1.0 is OK

Reply:
And this, too, when I deal with the rest of the fields. Thanks

Reply:
Done now!